integrations/airflow/tests/integration/docker/up.sh

```diff
@@ -52,4 +52,5 @@ marquez-python
 ${MARQUEZ_AIRFLOW_WHL}
 EOL
 
+docker-compose down
 docker-compose up --build --exit-code-from integration
```

Comment:
I would use the [`--force-recreate`](https://docs.docker.com/compose/reference/up/) flag to force recreating the containers if they are running.